algebra/src/curves/models/bls12/mod.rs

```diff
@@ -201,6 +202,7 @@ where
                 y5 *= &y4;
                 y5 *= &y1;
                 Some(y5)
-            } })
+            }
+        })
```

Comment:
Is there some option you used to prevent rustfmt from mauling this closure?  It normally screws up lines closed like `})`.

Reply:
Nope.